test/bench.cpp

```diff
@@ -534,7 +540,9 @@ TEST_CASE("benchmark tcp::server echo server thread pool", "[benchmark]")
         std::cerr << ms.count() << " : " << count << "\n";
     }
 }
+#endif
 
+/*
```

Comment:
Can we re-enable these tests?

Reply:
I just want to let you know that I have implemented all of your comments but this one because the test will not join the server threads.
I will continue to try to find the issue with the test in the following days. I will also push a commit with all the changes I made so far for another round of review. Thanks so far.